Source/Core/DolphinQt/Config/HardcoreWarningWidget.cpp

```diff
@@ -35,7 +35,7 @@ void HardcoreWarningWidget::CreateWidgets()
   auto* icon = new QLabel;
   icon->setPixmap(warning_icon);
 
-  m_text = new QLabel(tr("This feature is disabled in hardcore mode."));
+  m_text = new QLabel(tr("Unapproved codes will not be applied in hardcore mode."));
```

Comment:
In my opinion, this would be clearer without the double negative:

```suggestion
  m_text = new QLabel(tr("In hardcore mode, only approved codes will be applied."));
```

But this is only a suggestion.